include/nil/actor/zk/snark/systems/plonk/placeholder/proof.hpp

```diff
@@ -32,34 +32,42 @@ namespace nil {
     namespace actor {
         namespace zk {
             namespace snark {
+                constexpr std::size_t FIXED_VALUES_BATCH = 0;
+                constexpr std::size_t VARIABLE_VALUES_BATCH = 1;
+                constexpr std::size_t PERMUTATION_BATCH =2;
+                constexpr std::size_t QUOTIENT_BATCH = 3;
+                constexpr std::size_t LOOKUP_BATCH = 4;
 
                 /**
-                 * A proof for the Placeholder cheme.
+                 * A proof for the Placeholder scheme.
                  *
                  * While the proof has a structure, externally one merely opaquely produces,
                  * serializes/deserializes, and verifies proofs. We only expose some information
                  * about the structure for marshalling purposes.
                  */
                 template<typename FieldType, typename ParamsType>
                 struct placeholder_proof {
+                    static constexpr std::size_t FIXED_VALUES_BATCH = 0;
```

Comment:
I think here also enum preferable. Also if we define them in `actor::zk::snark` namespace - we can reuse them here.

Reply:
These are the same values, we are supposed to remove the upper one. Will do this sometime later.